Assets/SampleScenes/Menu/Scripts/MenuSceneLoader.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using UnityEngine;
+
+public class MenuSceneLoader : MonoBehaviour
+{
+    public GameObject menuUI;
+
+    private GameObject m_Go;
```

Comment:
please avoid this type of data naming

Reply:
This is not my file, it's standard assets for Unity.